diff -urN MHonArc2.4.2.orig/mha-dbedit MHonArc2.4.2/mha-dbedit
--- MHonArc2.4.2.orig/mha-dbedit	Thu Aug 12 17:37:46 1999
+++ MHonArc2.4.2/mha-dbedit	Thu Aug 12 17:38:25 1999
@@ -33,7 +33,7 @@
 ##---------------------------------------------------------------------------##
 
 MAIN: {
-    unshift(@INC, 'lib');	# Should I leave this line in?
+    unshift(@INC, '/usr/share/MHonArc');	# Should I leave this line in?
 
     ## Initialize MHonArc
     require 'mhamain.pl' || die qq/ERROR: Unable to require "mhamain.pl"\n/;
--- MHonArc-2.6.19/mha-dbrecover.orig	2024-04-20 07:24:49.052203467 +0200
+++ MHonArc-2.6.19/mha-dbrecover	2024-04-20 07:28:10.667777889 +0200
@@ -35,7 +35,7 @@ use Getopt::Long;
 ##---------------------------------------------------------------------------##
 
 MAIN: {
-    unshift(@INC, 'lib');	# Should I leave this line in?
+    unshift(@INC, '/usr/share/MHonArc');	# Should I leave this line in?
 
     ## Grab options from @ARGV unique to this program
     my %opts = ( );
--- MHonArc-2.6.19/mha-decode.orig	2024-04-20 07:24:49.052203467 +0200
+++ MHonArc-2.6.19/mha-decode	2024-04-20 07:28:37.270967100 +0200
@@ -35,7 +35,7 @@ use Getopt::Long;
 ##---------------------------------------------------------------------------##
 
 MAIN: {
-    unshift(@INC, 'lib');	# Should I leave this line in?
+    unshift(@INC, '/usr/share/MHonArc');	# Should I leave this line in?
 
     ## Grab options from @ARGV unique to this program
     my %opts = ( );
diff -urN MHonArc2.4.2.orig/mhonarc MHonArc2.4.2/mhonarc
--- MHonArc2.4.2.orig/mhonarc	Thu Aug 12 17:37:46 1999
+++ MHonArc2.4.2/mhonarc	Thu Aug 12 17:39:20 1999
@@ -32,7 +32,7 @@
 ##---------------------------------------------------------------------------##
 
 MAIN: {
-    unshift(@INC, 'lib');	# Should I leave this line in?
+    unshift(@INC, '/usr/share/MHonArc');	# Should I leave this line in?
 
     require 'mhamain.pl' || die qq/ERROR: Unable to require "mhamain.pl"\n/;
     mhonarc::initialize();
